components/builder-jobsrv/src/scheduler_datastore/test.rs

```diff
@@ -0,0 +1,358 @@
+#[cfg(test)]
+#[cfg(feature = "postgres_tests")]
+// cargo test --features postgres_tests to enable
+// from root
+// cargo test -p habitat_builder_jobsrv --features=postgres_tests
+// --manifest-path=components/builder-jobsrv/Cargo.toml
+mod test {
+    use crate::{assert_match,
+                data_store::DataStore,
+                hab_core::package::PackageTarget,
+                test_helpers::*};
+
+    use habitat_builder_db::{datastore_test,
+                             models::{jobs::{JobExecState,
+                                             JobGraphEntry,
+                                             JobStateCounts,
+                                             NewJobGraphEntry},
+                                      package::BuilderPackageTarget}};
+    use std::str::FromStr;
```

Comment:
You could probably replace most of these with `use super::*;`, as in "normal" testing modules.

Reply:
This is fallout of the weird test::test nesting; I couldn't figure out how to use super::super::* or whatever to bring things in.

Reply:
In `scheduler_datastore.rs` you'll want this:

```rust
#[cfg(test)]
#[cfg(feature = "postgres_tests")]
mod test;
```
Then, in `scheduler_datastore/test.rs`, you'll have this:

```rust
use super::*;

#[test]
fn do_a_thing() {
    // a thing
}
```
Only use the `super::*` if you need to import things you've already imported (or created) in `scheduler_datastore.rs`. It's exactly like you declared your test module in the `scheduler_datastore.rs` file.